tests/utils/component_test_utils.go

```diff
@@ -0,0 +1,233 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+
+	schema "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+	"github.com/google/go-cmp/cmp"
+	"sigs.k8s.io/yaml"
+)
+
+// AddVolume returns volumeMounts in a schema structure based on a specified number of volumes
+func AddVolume(numVols int) []schema.VolumeMount {
```

Comment:
func can be unexported, used only within the pkg

pls check for all funcs in this file.

Reply:
Done. Envisioned a time i the future when tests may get this specific and need these apis but unexported for now.